### PR TITLE
Add getExplorerLink()

### PR DESCRIPTION
**Pull request by mikemaccana**
- add: getExplorerLink()
- add: function options to headers in README
- add: whole squad to authors

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -28,6 +28,44 @@ export const getCustomErrorMessage = (
...
+  // https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
+  const url = new URL(baseUrl);
+  url.search = new URLSearchParams(searchParams).toString();
+  return url.toString();
+};
+
+export const getExplorerLink = (
+  linkType: "transaction" | "tx" | "address" | "block",
+  id: string,
+  cluster: Cluster | "localnet" = "mainnet-beta",
```
Comment (nickfrosty) [inline src/index.ts:44]: Bit of an edge case, but should we add support for other custom cluster urls?
Someone might have a specific url or part they are want to access a cluster

File: README.md [hunk 2]
```diff
@@ -99,16 +99,46 @@ const newBalance = await requestAndConfirmAirdropIfRequired(
...
-### getKeypairFromFile()
+### getKeypairFromFile(filename)
 
 Gets a keypair from a file - the format must be the same as [Solana CLI](https://docs.solana.com/wallet-guide/file-system-wallet) uses, ie, a JSON array of numbers:
 
 To load the default keypair `~/.config/solana/id.json`, just run:
 
 ```typescript
-const keyPair = await getKeypairFromFile();
+const keyPair = await getKeypairFromFile([file]);
```
Comment (nickfrosty) [inline README.md:141]: the `getKeypairFromFile` accepts a string string arg, not an array

File: README.md [hunk 3]
```diff
@@ -123,15 +153,15 @@ or using home dir expansion:
...
+### getKeypairFromEnvironment(environmentVariable)
 
 Gets a keypair from a secret key stored in an environment variable. This is typically used to load secret keys from [env files](https://stackoverflow.com/questions/68267862/what-is-an-env-or-dotenv-file-exactly).
 
 ```typescript
 const keyPair = await getKeypairFromEnvironment("SECRET_KEY");
 ```
 
-### addKeypairToEnvFile()
+### addKeypairToEnvFile(keypair, environmentVariable, [file])
```
Comment (nickfrosty) [inline README.md:164]: the `addKeypairToEnvFile` takes a string string as that last arg, not an array

**Review by nickfrosty — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mikemaccana) [reply README.md:141]: Good spot. I have no idea why I wrote that, guess that's why we have PRs.

File: README.md (hunk 3 above)
Comment (mikemaccana) [reply README.md:164]: Thanks again.

File: src/index.ts (hunk 1 above)
Comment (mikemaccana) [reply src/index.ts:44]: Interesting thought. Specifying values gets us validation in more cases, but yes if someone is using a custom cluster URL TS compilation will fail. Does that change anything for you? I could go either way so LMK what you prefer.

Comment (nickfrosty) [reply src/index.ts:44]: Yeah, that's fair. its an edge case like I mentioned.
Let's not worry about it for now. Worst case, we come back to it later if people start requesting it :)

**Pull request merged**